app/views/parks/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if params[:search].present? %>
```

Comment:
It's weird that you render one or the other on this page. Should `search` always be rendered but `results` be conditional?